### PR TITLE
update migration guide with dataset_ids and cdids params

### PR DESCRIPTION
**Pull request by Oluwawunmi**
### What

update migration guide with dataset_ids and cdids param filters.

### How to review

Check is ok and understandable.

### Who can review

Anyone.

### CODE REVIEW

File: static/retirement/v0api/index.md [hunk 1]
```diff
@@ -40,7 +41,8 @@ You can also browse the [full Search API Swagger specification](../../search/sea
 
 ## Migration guide
 
-Below you can find detailed ways to access the same data from the ONS using our latest services which will benefit from continued enhancement and investment.
+Below you can find detailed ways to access the same data from the ONS using our latest services which will benefit from
+continued enhancement and investment.
```
Comment (lindenmckenzie) [inline static/retirement/v0api/index.md:45]: Have you run this and seen if the line break is parsed?

File: static/retirement/v0api/index.md [hunk 2]
```diff
@@ -74,30 +77,40 @@ This endpoint currently returns a 404 for all `dataset_id` provided and so can a
 In order to get at similar data, you could still use our Search API:
 
 ```txt
-https://api.beta.ons.gov.uk/v1/search?content_type=dataset_landing_page&q={dataset_id}
+https://api.beta.ons.gov.uk/v1/search?content_type=dataset&dataset_ids={datasetID}
```
Comment (lindenmckenzie) [inline static/retirement/v0api/index.md:80]: We should be consistent about `dataset_id` or `datasetID`.

File: static/retirement/v0api/index.md [hunk 3]
```diff
@@ -132,9 +146,14 @@ See the [search response mapping](#search-response-mapping) for the overall resp
 
 There is not a direct replacement for this URI but the URIs for timeseries pages can be derived from search:
 
-`https://api.beta.ons.gov.uk/v1/search?q={timeseries_id}&content_type=timeseries`
+`https://api.beta.ons.gov.uk/v1/search?content_type=timeseries&cdids={cdid}`
+
+You can also filter by multiple timeseries IDs
```
Comment (lindenmckenzie) [inline static/retirement/v0api/index.md:151]:
```suggestion
You can also filter by multiple timeseries IDs:
```

File: static/retirement/v0api/index.md [hunk 4]
```diff
@@ -132,9 +146,14 @@ See the [search response mapping](#search-response-mapping) for the overall resp
 
 There is not a direct replacement for this URI but the URIs for timeseries pages can be derived from search:
 
-`https://api.beta.ons.gov.uk/v1/search?q={timeseries_id}&content_type=timeseries`
+`https://api.beta.ons.gov.uk/v1/search?content_type=timeseries&cdids={cdid}`
```
Comment (lindenmckenzie) [inline static/retirement/v0api/index.md:149]: Again, should be consistent about `cdid` vs `timeseries_id`

**Review by lindenmckenzie — COMMENTED** [review]
Few comments.

File: static/retirement/v0api/index.md (hunk 1 above)
Comment (Oluwawunmi) [reply static/retirement/v0api/index.md:45]: Doesn't break

File: static/retirement/v0api/index.md [hunk 5]
```diff
@@ -74,30 +77,40 @@ This endpoint currently returns a 404 for all `dataset_id` provided and so can a
...
 ### /dataset/{dataset\_id}/timeseries
 
-| Version | URI                                                                            |
-|---------|--------------------------------------------------------------------------------|
-| Old     | `{domain}/dataset/{dataset_id}/timeseries`                                     |
-| New     | `https://api.beta.ons.gov.uk/v1/search?content_type=timeseries&q={dataset_id}` |
+| Version | URI                                                                                     |
+|---------|-----------------------------------------------------------------------------------------|
+| Old     | `{domain}/dataset/{dataset_id}/timeseries`                                              |
+| New     | `https://api.beta.ons.gov.uk/v1/search?content_type=timeseries&dataset_ids={datasetID}` |
```
Comment (lindenmckenzie) [inline static/retirement/v0api/index.md:97]:
```suggestion
| New     | `https://api.beta.ons.gov.uk/v1/search?content_type=timeseries&dataset_ids={dataset_id}` |
```

File: static/retirement/v0api/index.md [hunk 6]
```diff
@@ -74,30 +77,40 @@ This endpoint currently returns a 404 for all `dataset_id` provided and so can a
...
 ### /dataset/{dataset\_id}/timeseries/{timeseries\_id} and /dataset/{dataset\_id}/timeseries/{timeseries\_id}/data
 
-| Version | URI                                                                               |
-|---------|-----------------------------------------------------------------------------------|
-| Old     | `{domain}/dataset/{dataset_id}/timeseries/{timeseries_id}`                        |
-| New     | `https://api.beta.ons.gov.uk/v1/search?content_type=timeseries&q={timeseries_id}` |
+| Version | URI                                                                          |
+|---------|------------------------------------------------------------------------------|
+| Old     | `{domain}/dataset/{dataset_id}/timeseries/{timeseries_id}`                   |
+| New     | `https://api.beta.ons.gov.uk/v1/search?content_type=timeseries&cdids={cdid}` |
```
Comment (lindenmckenzie) [inline static/retirement/v0api/index.md:107]:
```suggestion
| New     | `https://api.beta.ons.gov.uk/v1/search?content_type=timeseries&cdids={timeseries_id}` |
```

File: static/retirement/aggregatedsearch/index.md [hunk 7]
```diff
@@ -11,7 +11,7 @@ In line with our [approach to the retirement of API endpoints](../), it has been
 - strategic technology direction is elsewhere
 - cost of upkeep is too high with similar services available
 
-The UI for these pages has already been moved to use our Search API and it is the intention of ONS to separate data and UI endpoints by domain in future. Data in JSON format will be accessible via our API found at <https://api.beta.ons.gov.uk/v1/search>. You can also browse the [full API Swagger specification](../../search/search/).
+The UI for these pages has already been moved to use our Search API, and it is the intention of ONS to separate data and UI endpoints by domain in the future. Data in JSON format will be accessible via our API found at <https://api.beta.ons.gov.uk/v1/search>. You can also browse the [full API Swagger specification](../../search/search/).
```
Comment (lindenmckenzie) [inline static/retirement/aggregatedsearch/index.md:14]: I actually disagree with the additional 'the' in this sentence, but not enough to make you change it :D

File: static/retirement/v0api/index.md [hunk 8]
```diff
@@ -151,9 +170,10 @@ This endpoint currently returns a 404 for all `timeseries_id` provided and so ca
 
 To get a result to what might be expected for this endpoint, you can go to:
 
-`https://api.beta.ons.gov.uk/v1/search?q={timeseries_id}&content_type=timeseries`
+`https://api.beta.ons.gov.uk/v1/search?cdids={timeseries_id}&content_type=timeseries`
```
Comment (lindenmckenzie) [inline static/retirement/v0api/index.md:173]:
```suggestion
`https://api.beta.ons.gov.uk/v1/search?content_type=timeseries&cdids={timeseries_id}`
```

for consistency

**Review by lindenmckenzie — APPROVED** [review]

**Pull request merged**